tools/replay/parse_dump_resources_cli.cpp

```diff
@@ -78,51 +79,36 @@ static bool CheckIndicesForErrors(const gfxrecon::decode::VulkanReplayOptions& v
     // Indices should be provided sorted
     if (vulkan_replay_options.Draw_Indices.size())
     {
-        bool is_2d = false;
         for (const auto& indices : vulkan_replay_options.Draw_Indices)
         {
-            is_2d |= (indices.size() > 0);
-
             if (!AreIndicesSorted(indices))
             {
-                GFXRECON_LOG_ERROR("ERROR - incomplete --dump-resources parameters");
-                GFXRECON_LOG_ERROR("DrawCall indices are not sorted")
+                GFXRECON_LOG_ERROR("ERROR - incorrect --dump-resources parameters");
+                GFXRECON_LOG_ERROR("Draw indices are not sorted")
                 return true;
             }
         }
-
-        if (!is_2d)
-        {
-            GFXRECON_LOG_ERROR("ERROR - incomplete --dump-resources parameters");
-            GFXRECON_LOG_ERROR("DrawCall indices must be a 2 dimensional array")
-            return true;
-        }
     }
 
     if (vulkan_replay_options.RenderPass_Indices.size())
     {
-        bool is_3d = false;
+        bool is_complete = true;
```

Comment:
I don't think this is correct. If `vulkan_replay_options.RenderPass_Indices` is empty, `is_complete` will stay true and there won't be a warning

Reply:
RenderPass indices can be empty when doing a tracerays or dispatch dump, so we shouldn't give a warning then. Or am I wrong on RenderPass indices being empty on tracerays/dispatch?

Reply:
Right sorry. If `vulkan_replay_options.RenderPass_Indices` is empty the above if case will be false